src/analog/CommandBinaryBuffer.ts

```diff
@@ -1291,6 +1304,23 @@ class CommandBinaryBuffer extends BinaryBuffer {
         }));
     }
 
+    private getLegacyHourDiff (): ILegacyCounter {
+        const bytes = [this.getUint8(), this.getUint8()];
```

Comment:
```suggestion
        const [someName, anotherName] = [this.getUint8(), this.getUint8()];
```
?

Reply:
names added but don't know what they were for, no good variants for them